js/components/widgets/stackViewer/StackViewerComponent.js

```diff
@@ -508,8 +510,10 @@ define(function (require) {
         loadType: PIXI.loaders.Resource.LOAD_TYPE.IMAGE,
         xhrType: PIXI.loaders.Resource.XHR_RESPONSE_TYPE.BLOB
       };
+      let loaderResources = imageLoader.resources;
```

Comment:
@ddelpiano  Why assign here and not just on line 515?

Reply:
on line 515 I already lost the resources loaded due to the reset() call.